src/FileFormat/ImageStatistics.php

```diff
@@ -0,0 +1,26 @@
+<?php
+namespace CubeTools\CubeCommonBundle\FileFormat;
+
+/**
+ * Class handling statistics for image files (based on statistics for PDF files).
+ */
+class ImageStatistics extends PdfStatistics
+{
+    /**
+     * Method identifying image resolution. One image is one page.
+     *
+     * @param string $pageSize size in 1x1 format
+     * @param bool $returnPageSizeInMmIfNotMatched currently not implemented (default parameter)
+     *
+     * @return string image size in px (for example "100px x 100px")
+     */
+    public function identifyPageSize($pageSize, $returnPageSizeInMmIfNotMatched = true)
+    {
+        $pageSizeArray = explode('x', $pageSize);
+
+        return sprintf('%dpx x %dpx',
+                $pageSizeArray[0],
+                $pageSizeArray[1]
+                );
+    }
+}
```

Comment:
Why don't we put this and PdfStatistics into a single file?
Then https://github.com/EmchBerger/projectassistant/blob/19700cbff8a68a1224eedce6ccbfacd81b1015f5/src/AppBundle/CubePa/DocRepro/Processing/FileStatistics.php#L9 would be simpler or unneeded.

This file PdfImageStatistics/FileStatistics/PrintFileStatistics/... could identify the document type itself in `setFilename()` (what Processing/FileStatistics.php does now) and optionally get access to this property.

Reply:
The idea behind this separation was, that it is easier to read and change things, which are only for images or pdfs.